src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.top = '50%';
```

Comment:
Good job - it works! But now let's reimplement the task using element properties from the lesson (it will help you in the next task)